packages/widget.vite-plugin/src/plugin.ts

```diff
@@ -572,3 +589,107 @@ function extractWidgetConfig(objectExpression: ObjectExpression) {
 
   return JSON.parse(widgetConfigString);
 }
+
+function setWidgetSettings(
+  widgetRid: string,
+  foundryUrl: URL,
+  localhostUrl: string,
+  entrypointToJsSourceFileMap: Record<string, Set<string>>,
+  entrypointFileName: string,
+) {
+  const widgetDevModeSettings = {
+    entrypointJs: [
+      `/${VITE_INJECTIONS}`,
+      "/@vite/client",
+      ...entrypointToJsSourceFileMap[entrypointFileName],
+    ].map((file) => ({
+      filePath: `${localhostUrl}${file}`,
+      scriptType: { type: "module", module: {} },
+    })),
+    entrypointCss: [],
```

Comment:
Is CSS always injected by JS in dev mode?

Reply:
Yep